src/slot_machine.c

```diff
@@ -2395,41 +2663,60 @@ static void DecideReelTurns_NoBiasTag_Reel1(void)
     sSlotMachine->reelExtraTurns[0] = i;
 }
 
-static bool8 IsBiasTag777_SwitchColor(u8 *biasTagPtr)
+// If the bias tag is one of the 7's, switch to the opposite color and return
+// true. Otherwise, return false.
+static bool8 IfTag7_SwitchColor(u8 *tagPtr)
 {
-    if (*biasTagPtr == GFXTAG_7_RED)
+    if (*tagPtr == GFXTAG_7_RED)
```

Comment:
`ptr` is unnecessary in the name, this is evident from the type

Reply:
Done